pom.xml

```diff
@@ -294,6 +294,12 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-yarn_${scala.binary.version}</artifactId>
```

Comment:
why it needs to add spark-yarn jar?

Reply:
it's for testing on yarn. it's provided so it won't incur overhead